falcon/app.py

```diff
@@ -524,7 +524,21 @@ class can use suffixed responders to distinguish requests
                 for a single item vs. a collection of those same items.
                 Another class might use a suffixed responder to handle
                 a shortlink route in addition to the regular route for the
-                resource.
+                resource. For example::
+
+                    class Baz(object):
+
+                        def on_get_foo(self, req, resp):
+                            pass
+
+                        def on_get_bar(self, req, resp):
+                            pass
+
+                    baz = Baz()
+                    api = falcon.API()
```

Comment:
:arrow_right: `app = falcon.App()` here too, and we'd be done I think :sweat_smile: